.circleci/config.yml

```diff
@@ -61,11 +61,17 @@ jobs:
           name: e2e test
           command: |
             if [[ $CIRCLE_BRANCH != *"master"* || ! -z $CIRCLE_TAG ]] ; then
```

Comment:
you can move `set +e` to the top of the `command`, like

```
command: |
  set +e;
  if [[ $CIRCLE_BRANCH != *"master"* || ! -z $CIRCLE_TAG ]] ; then
    E2EPROFILE=saucelight npm run test:e2e;
  else
    npm run test:e2e;
  fi

   STATUS=$?
   if [[ ! -z $FORCE_BUILD ]] ; then
     echo "The step exited with $STATUS but FORCE_BUILD is active, so we mark it as passed";
     exit 0;
   else
     exit $STATUS;
   fi
```

Reply:
Done